svc-composition-service/app/config/config.py

```diff
@@ -0,0 +1,64 @@
+import os
+import json
+
+CONF_FILE = os.getenv("PLUGIN_CONFIG_FILE_PATH")
```

Comment:
As mentioned above, please modify your logic to see composition service as a service and not a plugin.
Use service config file, which does not need odim credentials, ideally, service would need certificates for rpc communication and DB details.